scss/colors.scss

```diff
@@ -0,0 +1,3 @@
+$grey: #C4C4C4;
```

Comment:
same that you do with colors you can do with sizes:

```
$text-sm: <enter small text size>
$text-md: <enter small text size>
$text-lg: <enter small text size>
```

Another thing we tend to put all this variables inside a folder name variables in the scss folder and we name the file like `_colors.scss`